docs/howto/ingest_geoip.md

```diff
@@ -2,7 +2,19 @@
 
 Elasticsearch provides default GeoIP databases that can be downloaded in runtime and which weights ~70 MB. This can be
 a root cause of flakiness of package tests, so elastic-package embeds small samples of GeoIP databases, that can identify
-accurately only few ranges of IP addresses included [here](../../internal/fields/_static/allowed_geo_ips.txt)
+accurately only few ranges of IP addresses included [here](../../internal/fields/_static/allowed_geo_ips.txt).
```

Comment:
We want to eventually use documentation IP ranges only. I think we can remove the mention to `allowed_geo_ips.txt` in docs.